oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -47,27 +49,70 @@ std::string DumpHandler::findDumpObjPath(uint32_t fileHandle)
         "xyz.openbmc_project.Dump.Manager";
     static constexpr auto DUMP_MANAGER_PATH = "/xyz/openbmc_project/dump";
 
+    static constexpr auto OBJECT_MANAGER_INTERFACE =
+        "org.freedesktop.DBus.ObjectManager";
+    auto& bus = pldm::utils::DBusHandler::getBus();
+
+    if (dumpType == PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS)
+    {
+        resDumpRequestDirPath = "/var/lib/pldm/resourcedump/" +
+                                std::to_string(fileHandle);
+    }
+
     // Stores the current resource dump entry path
     std::string curResDumpEntryPath{};
 
-    ObjectValueTree objects;
-    // Select the dump entry interface for system dump or resource dump
-    DumpEntryInterface dumpEntryIntf = systemDumpEntry;
+    if (dumpType == PLDM_FILE_TYPE_BMC_DUMP)
```

Comment:
Please remove if- else if conditions and add a single one

Reply:
done